Bottom: 3365e7f9e292693b892fe05f705c9491ed4be9a8
Top:    59607ce07ae3add61284f3184a699451e2326fab
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 22:46:40 -0500

Add Rakefile

---

diff --git a/Rakefile b/Rakefile
new file mode 100644
index 0000000..7b4c20b
--- /dev/null
+++ b/Rakefile
@@ -0,0 +1,9 @@
+desc "Remove trailing whitespace"
+task :rm_whitespace do
+  `sed -i 's/[[:space:]]*$//' script.js syntax.php`
+end
+
+desc "Reformat JS code"
+task :reformat_js do
+  `uglifyjs --beautify --output script.js -- script.js`
+end
